doc/glossary.rst

```diff
@@ -205,6 +205,23 @@ General Concepts
         exceptional behaviours on the estimator using semantic :term:`estimator
         tags`.
 
+    cross-fitting
+    cross fitting
+        A resampling method that iteratively partitions data into complementary
+        'train' and 'estimation' subsets. The 'estimation' subset may be used
+        to estimate a parameter or predict a target. The outputs from
+        each iteration are combined and used in a downstream prcess, generally
+        to avoid bias.
+
+    cross-validation
+    cross validation
+        A resampling method that iteratively partitions data into complementary
+        'train' and 'test' subsets so model performance can be evaluated on
+        unseen data. This conserves data as avoids the need to hold out a
+        'validation' dataset and accounts for variability as multiple rounds of
+        cross validation are genreally performed.
+        See :ref:`User Guide <_cross_validation>` for more details.
```

Comment:
There is a problem with this reference:

```
The following documentation warnings may have been generated by PR #https://github.com/scikit-learn/scikit-learn/pull/26677:
/home/circleci/project/doc/glossary.rst:200: WARNING: undefined label: '_cross_validation'
```

I believe there should be no leading underscore.

Reply:
Yes you are right, thanks. Bad copy paste